dist/angular-ui-notification.js

```diff
@@ -145,7 +160,7 @@ angular.module('ui-notification').provider('Notification', function() {
 
                 setCssTransitions('none');
 
-                angular.element(document.querySelector(args.container)).append(templateElement);
+                angular.element(document.getElementsByTagName('body')).append(templateElement);
```

Comment:
thank you for insight, but I havent changed this line by myself.   What i've done is dealing with Template loading. I'll just fix it as you've adviced. Thank you.